### PR TITLE
INT-414 Red Hat Certified Recipe

### PR DESCRIPTION
**Pull request by whyjustin**
https://issues.sonatype.org/browse/INT-414

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -32,6 +32,12 @@ suites:
     verifier:
       inspec_tests:
         - test/smoke/docker
+  - name: rhel-docker
```
Comment (bigspotteddog) [inline .kitchen.yml]: This is not clear at all and it needs to be added to the README somewhere. This needs to added to the .kitchen.yml.erb file to get Jenkins to test it. .kitchen.yml is used when testing locally.

The Jenkinsfile's Test stage takes the parameters passed in for key pair, security group, and subnet, then generates the .kitchen.yml file used for testing from .kitchen.yml.erb.

Comment (whyjustin) [reply .kitchen.yml]: 4517897

> This is not clear at all and it needs to be added to the README somewhere. 

Yes

File: test/smoke/rhel-docker/perm_test.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Permissions test for recipe nexus_repository_manager::rhel-docker
+
+control 'rhel-docker-perms-test-001' do
+  title 'Allows nexus user to mutate /etc/password'
+  describe user('nexus') do
+    it { should exist }
+  end
+
+  describe file('/etc/passwd') do
+    it { should be_allowed('write', by_user: 'nexus') }
```
Comment (bigspotteddog) [inline test/smoke/rhel-docker/perm_test.rb]: This test is failing. I am not sure why yet.

Comment (bigspotteddog) [reply test/smoke/rhel-docker/perm_test.rb]: Test is now passing after upgrading the ami used for chef testing with the latest chef version on jenkins.

**Pull request merged**